.changes/unreleased/Fixes-20231013-130943.yaml

```
@@ -0,0 +1,6 @@
+kind: Fixes
+For packages installed with tarball method, fetch metadata to resolve nested dependencies
```

on us - accidentally deleted the key!
```suggestion
body: For packages installed with tarball method, fetch metadata to resolve nested dependencies
```